### ISSUE TITLE
The limit of 10 atomic operations has been increased

### ISSUE THREAD

**Issue by inverted-capital** [kitsonk/kv-toolbox#8]
In the README.md file, reference is made to the limit of 10 mutations maximum.

I think this is possibly outdated ?  I can't find exact documentation of examples, but this commit indicates the limit is now on total transaction size being under 800kB, rather than the number of operations: https://github.com/denoland/deno/pull/20126/commits/28208bfd6dbfe78dbd7068e45625caf0e4076034

I'm hoping this greatly expands the atomic blob capabilities that kv-toolbox can enable for us - thank you once again for making this library 🙏

**Comment by kitsonk** [comment]
It appears the limit was changed from 10 to 1000 in https://github.com/denoland/deno/pull/20126, but it also introduced an overall byte limit per mutation. The commit you mention above was part of an abandoned fork it appears.

It appears now that there are a [few limits](https://github.com/denoland/deno/blob/524e451bfbd36d3f5a00a16a7115b31b56805f10/ext/kv/lib.rs#L71-L75) on an atomic transaction, but the value size is still 64K.  I will work to get batchedAtomic to work in line with those limits, but it is important to realise it doesn't really change much for blobs... They are still limited to 64k chunks, but at least they are unlikely to span a single atomic transaction once I have updated kv-toolbox.
